ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -1819,3 +1819,731 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
```

Comment:
This is all generated? Seems like quite a few unrelated jobs added here. Just want to confirm these are all intentional.

Reply:
This is all generated from the addition of ci-operator/config/openshift/release/openshift-release-master__ocp-4.7.yaml.  Steve said I could add it.  Maybe it is premature?